### PR TITLE
Positions

### PR DESCRIPTION
**Pull request by StephanieYHe**
Adding 4 cardinal positions option. 

Issue exists when a tooltip is moused over

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,48 @@
 bower_components/
+accessibility-developer-tools/
```
Comment (notwaldorf) [inline .gitignore]: Oh, this shouldn't be needed. I'm not sure how your project is set up, but an element doesn't live in the same directory as the other elements unless you specifically made it so -- which means this kind of .gitignore doesn't live here :(

File: paper-tooltip.html [hunk 2]
```diff
@@ -70,10 +70,46 @@
 
       @apply(--paper-tooltip);
     }
+
+    #arrow {
```
Comment (notwaldorf) [inline paper-tooltip.html]: This is not part of the Material Design spec, and it was omitted on purpose, I'm afraid.

**Comment by notwaldorf** [comment]
Woohoo! That's exactly what I was going to start on next, and now I don't have to! Thanks for the contribution! I left a couple of initial comments, but I'll take a full look at it tomorrow! :)

**Comment by notwaldorf** [comment]
@StephanieYHe Are you still working on this?

**Comment by StephanieYHe** [comment]
Hi Monica,

So sorry about the lack of an update! My most recent working version is
here:
https://github.com/fluxio/paper-tooltip/blob/position/paper-tooltip.html

I made it for the startup I'm at, and it still contains the arrows (which
is what we wanted, even though it's not part of the Material Design spec).
Would you want me to create another pull request with a branch where the
arrows are removed?

Let me know what I can do, and thanks for following up on this.

Cheers,
Stephanie

On Thu, Aug 13, 2015 at 5:11 PM, Monica Dinculescu <notifications@github.com

**Comment by notwaldorf** [comment]
Sure, if you want to. Otherwise, I can just pick up this branch and clean it up and merge it. Depends on if you have time for it :)

**Comment by notwaldorf** [comment]
I've created this PR based on yours, so I'm going to close this one -- there were a couple of things that needed fixes, and I'm going to add test.

Thanks so much for the help!

**Comment by StephanieYHe** [comment]
Glad it could be helpful. :)

On Fri, Aug 14, 2015 at 5:05 PM, Monica Dinculescu <notifications@github.com

**Pull request closed without merging**